hudi-cli/src/main/java/org/apache/hudi/cli/commands/SparkMain.java

```diff
@@ -148,7 +148,7 @@ public static void main(String[] args) throws Exception {
           }
           configs = new ArrayList<>();
           if (args.length > 9) {
-            configs.addAll(Arrays.asList(args).subList(8, args.length));
+            configs.addAll(Arrays.asList(args).subList(9, args.length));
```

Comment:
nit: have a constant variable for `9`?

Reply:
looks like there are many instances of this pattern.  I created a Jira for clean-up: [HUDI-4228](https://issues.apache.org/jira/browse/HUDI-4228)